src/NuGet.Core/NuGet.Configuration/Settings/Items/ClientCertItem.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Security.Cryptography.X509Certificates;
+using System.Xml.Linq;
+
+namespace NuGet.Configuration
+{
+    public abstract class ClientCertItem : SettingItem
+    {
+        public static int MergeHash(params int[] hashes)
```

Comment:
I think we have hash code combiner that can be used isntead of this.

Reply:
replaced with right one